<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayStudent</class>
 <widget class="QPushButton" name="DisplayStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
  </property>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Student Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShiftcount">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string># of shifts assigned</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShiftMax">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>(# max shifts)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
